utils/verify_wav_stimuli.ipynb

```diff
@@ -244,6 +244,39 @@
     "    print('Your stimuli are sufficiently damped, '\n",
     "          'you are using {:.1f}% of the range.'.format(100./att_factor))"
    ]
+  },
```

Comment:
fint plot, men måske skal den længere oppe, for at motivere hvorfor hele denne notebook er nødvendigt?